src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -956,9 +961,8 @@ void Transformations::runLptPasses(const std::vector<ov::element::Type>& default
         AddTransformation);
     CPU_DISABLE_PASS_COMMON(lptManager, MultiplyToGroupConvolutionTransformation);
 
+    CPU_DISABLE_PASS_ARM(lptManager, FakeQuantizeTransformation);
```

Comment:
It's better to avoid `FakeQuantizeTransformation` disabling at all. If we need to avoid this transformation in some specific cases (to preserve convolution dequantization scale as separate op), it is better to regulate this via transformation callback